src/Resources/Wacc/Orders/Orders.ts

```diff
@@ -40,4 +40,9 @@ export default class Orders extends AbstractResource {
         method: 'GET',
         path: '/pick',
     });
+
+    public assignOrders: Function = this.createMethodFromPartialSpec({
```

Comment:
I think we are meant to use the new syntax for this. Check the `/src/Resources/Subscriptions/PricingPlans.ts` file for an example of it